.rspec_parallel

```diff
@@ -0,0 +1,4 @@
+--no-profile
+--format-progress
+--format RspecJunitFormatter
+--out rspec-output/rspec-output<%= ENV['TEST_ENV_NUMBER'] %>.xml
```

Comment:
Is this going to be the default locally? If so, that's going to be a bit of a PITA since it'll generate `RspecJunitFormatter`, no? Is there a way to specify a config file specific to CI?

Reply:
I guess one way could be to add the output to `.gitignore`
On previous projects we would have a `CI` environment so that we could create config specifically for this. 
@rizzkhan1 - do you have any ideas?

Reply:
What I was worried about is that when we run `rspec` in parallel locally, it'll output in `RspecJunitFormatter` and not in the usual format we're used to. Is that not the case?

Also, ya, we'd need to ignore the output file to make sure it doesn't get checked in.

Reply:
Ok, looks like the default `progress` format is enabled in this file, I missed that. However, I'm not sure that configuring it here is the right thing to do. This output is only needed for CI, so why are we enabling it everywhere? Shouldn't we just specify the `--format` options in the Azure Pipeline steps, since this is specific to Azure?

Reply:
I say we rename this file so that it doesn't get used by default locally, and also change `--format progress` to `--format documentation`. That would bring us closer to what we had before (it used to cat out the XML file).